ndx-depth-moseq/ndx-meta.yaml

```
@@ -0,0 +1,8 @@
+name: ndx-depth-moseq
+version: 0.1.1
```

```suggestion
version: 0.1.2
```

Update to latest version after it is released on PyPI